pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -284,7 +284,6 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 		}
 	} else {
 		// Pod is not present, create pod.
-		go c.timeoutHandler.WaitTaskRun(tr)
```

Comment:
This line is duplicated with [this one](https://github.com/tektoncd/pipeline/pull/691/files#diff-b52b9f382ad878e6aa2a85921bb19065L307) — see https://github.com/tektoncd/pipeline/pull/674 for reference